Interview/ConductingATechnicalInterview.md

```diff
@@ -0,0 +1,24 @@
+Outline:
+
+How to Conduct a Technical Interview
+====================================
+
+### Remember what You Want to Achieve
+Best possible outcome from an interview is a new hire who makes great contributions to your organisation, your team and thrives as a member in your team. In this section I will try to say something about the mindset required to interview a candidate successfully.
+
+### Be prepared
+To not prepare for an interview is simply rude. How to best prepare does, however, depend on the kind of interview.
+
+### Make the Candidate Feel Welcome
+Discuss what we can do to make a probably nervous candidate feel welcome and respected.
+
+### Look for what the Candidate can bring
+Diverse teams are well known to be more productive but at the same time a new hire should enough about the technology we use for their intended level. How can one think about structuring an interview to learn something about what the candidate brings that we don't have already while not forgetting to ask about essential skills.
+
+###  Let the Candidate have the Floor
+- It is not your interview
+- It is not your chance to take a step forward in your career
+- Showing off your skills to a candidate is never going to earn you a pay rise or promotion. Save your boasting for your next one-to-one with your line manager.
+
+### Think about Our Brand
+You are representing Babylon when conducting an interview. Here I want to discuss what we can do to make sure that a candidate always leaves with a positive impression of Babylon.
```

Comment:
👍  💯 
I like this section: It's often forgotten when conducting interviews that this is a two-sided thing. The Candidate has to demonstrate that they're fit to join the team, but we as a team also have to show them that Babylon is a good team and environment that they'll like 🙂